client/src/components/LeftSidebar/components/Categorical/components/AddLabelDialog.tsx

```diff
@@ -0,0 +1,122 @@
+import React from "react";
```

Comment:
Dialog note: the add/rename flows use the shared `LabelInput` component but now dispatch through the new annotation thunks so the autosave baseline is updated immediately after the modal closes.